### DISCUSSION TITLE
Polish translation

### DISCUSSION THREAD

**Discussion by WojciechSkirlo** [vuejs-translations/guidelines#31] [Ideas]
Hi there! I thought to create polish docs for Vue 3. I want to popularize this framework in my country and translation to polish could help in this. 

Page: https://polite-cranachan-4a25ff.netlify.app/
Repo: https://github.com/WojciechSkirlo/docs

**Comment by dawidow** [comment]
Yes, this is something we Poles need. I'm all for it, and I'll help if I have to.

**Reply by andrzejrumak** [reply]
Is it good idea? I have been thinking about it for about 2 year. What is better for Polish programmers learning English and have opened doors to the world and learn frameworks or programming language in English or use Polish sources of learning? Anyway I want to help if it would be useful. Finally I think I can learn more and help other make progress faster in VUE learning. Regards.

**Reply by WojciechSkirlo** [reply]
So yeah, we can get to work. I'll write some issues on github to better organization.

**Reply by andrzejrumak** [reply]
You know I am not a profesional programmer but for 5 years learning and use Vue. I want to help to translate. How I can help? What is the easiest for me? How I can start in the right way.

**Reply by WojciechSkirlo** [reply]
Clone repository on the computer and then create own branch. For example, if a task on GitHub is named Feature | Introduction you create a branch named feature/introduction and then translate the page and push the changes on the server then I check it out and if everything is ok I merge it with the main branch.
I assigned you already to one issue on GitHub. If you have any questions ask me.

**Comment by andrzejrumak** [comment]
Thanks for the explanation. I have just cloned repo. One question What do
you think Is it needed to translate technical words - for example **Declarative
Rendering**: or **Reactivity**
I am going to create my branch and translate it tomorrow. Do you want me to
translate all the introductions and push the changes or I can do it
partly?
Andrzej

pt., 6 maj 2022 o 17:45 Wojciech Skirło ***@***.***>
napisał(a):

**Reply by WojciechSkirlo** [reply]
For now we can translate it partly. Maybe in the future if it will be needed we can think about it.

**Comment by WojciechSkirlo** [comment]
Cool, from now on I will be adding new issues on GitHub and if you will have time to translate something you just assign the task to yourself so that I would know to take another task

**Reply by andrzejrumak** [reply]
OK If my translation is OK I am going to translate #4 then in the same way.

**Reply by andrzejrumak** [reply]
Sorry It is first time doing this so I saw that you are assigned to #4 and I started the new issue. But finally start new issues, please by yourself and I will make assignment for. Now Is have created new brunch feature/conditionalrendering and I will be translating it evening today. Have a good day.

**Reply by WojciechSkirlo** [reply]
No problem nothing happened. Have a nice day

**Reply by andrzejrumak** [reply]
I have finished feature/conditionalrendering translation. Could you check it and merge, please. What should I do next? Do you know how much should be translated that the Polish translation can be visible on the main Vue page?

**Reply by WojciechSkirlo** [reply]
I merged branches. Take another task from the issues. I've got no idea how much we have to translate. If we have translated around 80% we can think about it.

**Comment by andrzejrumak** [comment]
Hello
I have just finished template syntax. You can check and merge it.
Andrzej

wt., 10 maj 2022 o 07:27 Andrzej Rumak ***@***.***> napisał(a):

**Reply by WojciechSkirlo** [reply]
Merged

**Comment by andrzejrumak** [comment]
Hi,
I have just started Reactivity Fundamentals and What do you think what is
better translation for "option" - opcja czy parameter if we think about
Options API - for example
W Options API używamy parametru `data`, aby zadeklarować reaktywny stan
komponentu.
or
W Options API używamy opcji `data`, aby zadeklarować reaktywny stan
komponentu.
????

Have a good day.


wt., 10 maj 2022 o 23:09 Andrzej Rumak ***@***.***> napisał(a):

**Reply by WojciechSkirlo** [reply]
Osobiście wybrałbym to tłumaczenie:
`W Options API używamy parametru data aby zadeklarować reaktywny stan
komponentu.`

Ale równie dobrze zamiast tego można by było napisać 
`W Options API używamy funkcji data, aby zadeklarować reaktywny stan
komponentu.`

**Comment by andrzejrumak** [comment]
And how translate "Hooks" or maybe it should not be translated?
Regards
Andrzej

śr., 11 maj 2022 o 07:43 Andrzej Rumak ***@***.***> napisał(a):

**Reply by WojciechSkirlo** [reply]
Ja bym tego chyba nie tłumaczył.

**Comment by andrzejrumak** [comment]
Thanks for advice, checking and merge. I hope you've read before merging.
Tomorrow I will continue.

śr., 11 maj 2022, 20:19 użytkownik Wojciech Skirło ***@***.***>
napisał:

**Comment by andrzejrumak** [comment]
I have just finished Creating Vue App and I am going to translate Template
syntax in the evening.
Could you work on the next issues, please.
Have a good day.
Andrzej

pon., 9 maj 2022 o 21:02 Wojciech Skirło ***@***.***>
napisał(a):

**Comment by Mensix** [comment]
Hi, do you need some help with translating docs? I have seen some sections that sound a bit weird and I would like to clean them up.

**Reply by WojciechSkirlo** [reply]
Of course. You can help us. I added you to the repository.

**Reply by andrzejrumak** [reply]
@Mansix Thanks for help. Now I am very busy but I am going to continue in 2 week.

**Comment by WojciechSkirlo** [comment]
I synced our repository with the main repository. A lot of things have changed so we have to change some translations again.

Have a good day

**Comment by szymonlicau** [comment]
Cześć!

It has been a while since some last updates here. I'd like to help make this a reality here and I'm not the only one - our Vue team at Monterail has decided to take up the efforts of making the Polish translation a reality.

However, seeing this initiative kind of stopped more than a year ago and there is now quite a big chunk of commits unsynced to the repository where this initiative started - we figured out to start from scratch and start making the translations on a fresh copy as we don't really have a good idea what was done or not and how things have changed since the last commits.

FYI, the repo is here: https://github.com/monterail/vue3-docs-pl

There isn't much work done there yet, as we have just started making this initiative a reality internally but feel free to check it out and help if you would like to!

**Comment by szymonlicau** [comment]
Hello @Jinjiang.

I think we could start the process of moving the docs repo (https://github.com/monterail/vue3-docs-pl). We are about 80-90% done and most of the things remaining are in progress and looking to be finished soon (or are very tiny details). I've just detached the repo as well.
Let me know if we need to do anything else, thanks!

**Reply by Jinjiang** [reply]
@szymonlicau it looks great! I saw the repo has been well-detached. I've sent you an invitation to join our org. And then you can feel free to transfer the repo.
After the transfer, we will setup the official deploy and domain as well.
Thanks.

**Reply by szymonlicau** [reply]
Thanks @Jinjiang!

I have also now transferred the repo so it's available here: https://github.com/vuejs-translations/docs-pl.

**Reply by Jinjiang** [reply]
Great job! Now it's officially hosted https://pl.vuejs.org/ 🎉

**Reply by Jinjiang** [reply]
BTW, PRs are also welcome to update your translation progress widely
https://github.com/vuejs/docs/edit/main/src/translations/index.md

**Reply by szymonlicau** [reply]
Totally forgot about that, I'll make a PR shortly 😄
